tests/test_manager_api.py

```diff
@@ -121,9 +117,9 @@ def test_unknown_app(method_name_args):
     invalid_app_id = "oops_no_such_app"
     dapp = DappManager(invalid_app_id)
     with pytest.raises(UnknownApp) as exc_info:
-        #   NOTE: we have both properties and methods here, exceptions for properties are
-        #         raised in getattr, for methods when they are executed (*args), but both is fine
-        #         so this doesn't really matter
+        # NOTE: we have both properties and methods here, exceptions for properties are
+        #  raised in getattr, for methods when they are executed (*args), but both is
```

Comment:
as far as I understand, this should be plural?

```suggestion
        #  raised in getattr, for methods when they are executed (*args), but both are
```